src/dev/DirectManipulationHelper.js

```diff
@@ -0,0 +1,190 @@
+﻿import React, { useCallback, useEffect, useState, useMemo } from 'react';
+import { StyleSheet, Text, View, UIManager, SectionList, YellowBox, FlatList, TouchableOpacity, ScrollView, TextInput } from 'react-native';
+import ReanimatedModule from '../ReanimatedModule';
+import * as _ from 'lodash';
```

Comment:
By optional what do you mean?
Isn't `lodash` part of `react-native`'s deps?
Notice that this component isn't a production component, is required inline and renders only in `__DEV__` mode (I was thinking that `lodash` would be required from the parent package).

This might be the cause for the metro bundling error BTW.

Reply:
I committed a change to `metro.config.js`